x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/NodeDeprecationChecks.java

```diff
@@ -41,7 +44,7 @@
 
 class NodeDeprecationChecks {
 
-    static DeprecationIssue checkPidfile(final Settings settings, final PluginsAndModules pluginsAndModules) {
+    static DeprecationIssue checkPidfile(final Settings settings, final PluginsAndModules pluginsAndModules, ClusterState cs) {
```

Comment:
Nit, for consistency I'd prefer this (and similarly below too)

```suggestion
    static DeprecationIssue checkPidfile(final Settings settings, final PluginsAndModules pluginsAndModules, final ClusterState clusterState) {
```

I'm ok with `state` instead of `clusterState` although I prefer the latter.

Reply:
Yeah, I agree, I was too lazy.....